backend/src/controllers/dish.ts

```diff
@@ -100,29 +74,70 @@ export const getDishes = async (req: Request, res: Response) => {
             message: 'Error when mapping dishes to transactions',
             statusCode: 500,
         })
-        res.status(500).json({ error: 'internal_server_error' })
-        return
+        return res.status(500).json({ error: 'internal_server_error' })
+    }
+
+    let userDishes
+    if (withTransactions !== 'true') {
+        try {
+            if (borrowed == 'true') {
+                userDishes = getAllUserDishesInUse(userClaims, allDishes, dishTransMap)
+            } else {
+                userDishes = getAllUserDishes(userClaims, allDishes, dishTransMap)
+            }
+        } catch (e) {
+            Logger.error({
+                error: e,
+                message: 'error when getting user dishes',
+                statusCode: 500
+            })
+            return res.status(500).json({ error: 'internal_server_error' })
+        }
+        return res.status(200).json({ userDishes })
```

Comment:
Can we keep consistent naming?
```suggestion
        return res.status(200).json({ dishes: userDishes })
```